src/components/data-management/SamplesTableCells.jsx

```diff
@@ -47,6 +53,7 @@ const UploadCell = (props) => {
     setUploadDetailsModalData({
       sampleUuid,
       fileCategory: columnId,
+      lastModified: sample.lastModified,
```

Comment:
Bugfix - this was not passed before, therefore showing wrong result for the last modified date